manager/controlapi/common.go

```diff
@@ -94,8 +98,17 @@ func validateDriver(driver *api.Driver, pg plugingetter.PluginGetter, pluginType
 		return nil
 	}
 
-	p, err := pg.Get(driver.Name, pluginType, plugingetter.Lookup)
+	if pg != nil {
+		p, err = pg.Get(driver.Name, pluginType, plugingetter.Lookup)
+	} else {
+		_, err = plugins.Get(driver.Name, pluginType)
```

Comment:
Hi, @aaronlehmann thanks for the review.

- The reason I add `plugins.Get(driver.Name, pluginType)` is to mimic the output of docker deamon like 
```
WARN[0000] Unable to locate plugin: newdriver, retrying in 1s 
WARN[0001] Unable to locate plugin: newdriver, retrying in 2s 
WARN[0003] Unable to locate plugin: newdriver, retrying in 4s 
WARN[0007] Unable to locate plugin: newdriver, retrying in 8s 
```
Otherwise, we can return the error code to the client directly.

- You are right; I will add something like `if p != nil` before calling `p.IsV1`.